<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPBrushPresetsWidget</class>
 <widget class="QWidget" name="MPBrushPresetsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mypaint brush presets</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="addPresetButton">
       <property name="text">
        <string>Add preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePresetButton">
       <property name="text">
        <string>Remove preset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="presetsListView"/>
   </item>
   <item>
    <widget class="QPushButton" name="resetPresetsButton">
     <property name="text">
      <string>Reset to default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
